numba_dpex/tests/dpjit_tests/dpnp/test_dpnp_full_like.py

```diff
@@ -148,18 +148,34 @@ def func(x, fill_value, queue):
     assert c.dtype == dtype
     assert c.usm_type == usm_type
     assert c.sycl_device == queue.sycl_device
-
-    if c.sycl_queue != queue:
-        pytest.xfail(
-            "Returned queue does not have the same queue as the one passed to the dpnp function."
-        )
+    assert c.sycl_queue == a.sycl_queue
```

Comment:
Same here. Are we supposed to have these two statements true at the same time?

Reply:
Yes